compose/project.py

```diff
@@ -215,6 +221,22 @@ def build(self, service_names=None, no_cache=False, pull=False, force_rm=False):
             else:
                 log.info('%s uses an image, skipping' % service.name)
 
+    def create(self, service_names=None, strategy=ConvergenceStrategy.changed, do_build=True):
+        services = self.get_services_without_duplicate(service_names, include_deps=True)
+
+        plans = self._get_convergence_plans(services, strategy)
+
+        for service in services:
+            plan = plans[service.name]
+            if plan.action == 'create':
+                service.create_container(do_build=True)
+            elif plan.action == 'recreate':
+                service.recreate_container(
+                    service.containers(stopped=True)[0],
+                    do_build=do_build,
+                    start_new_container=False
+                )
```

Comment:
I think it would be good to keep this logic in `service.py`, since it's very similar to the existing `execute_convergence_plan`. Maybe `service.execute_create_convergence_plan()` ? It's a bit wordy, but at least it's clear.